server/liveblog/themes/themes_assets/default/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "liveblog-default-theme",
-  "version": "3.9.0",
+  "version": "3.9.0-dev.7",
```

I think this should be
```suggestion
  "version": "3.90.0-dev.7",
```